tests/test_kv_config_store.py

```
@@ -128,15 +141,205 @@ class _UnavailableConfigBackend:
+    Devin-review finding on #1017: only the ``get_config_store()`` factory ran
+    the legacy-category backfill, so a caller building its own store (e.g. a
+    real Postgres-backed one already carrying persisted ``routing.<key>`` rows)
+    and injecting it straight into ``CostRoutingCoordinator`` never got it
+    migrated -- ``RoutingPolicy`` and ``build_job_registry`` would then silently
+    fall back to their hardcoded defaults for that legacy data. Complements
+    ``tests/test_routing_config_compatibility.py``'s focused
+    ``RoutingPolicy``-level unit test with a full ``CostRoutingCoordinator``
+    integration check, since that is the real production entry point an
+    injected store actually arrives through.
+    """
+    legacy_category, (replacement_category, config_keys) = next(
```

Ruff (0.16.3)

[warning] Unpacked variable `config_keys` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)